### ISSUE TITLE
<PhoneInput listPortalTarget={document.body}> doesn't use LightScheme

### ISSUE THREAD

**Issue by jahe** [freenowtech/wave#434]
The country code selection menu of `<PhoneInput listPortalTarget={document.body}>` appears with dark scheme colors although `<LightScheme>` is used.

In the documentation it's not reproducible because `light-scheme` is set on `<body>`. In our case `light-scheme` is set on the root `<div>`. Problem is that the selection menu of `<PhoneInput>` is being rendered right under `<body>`. 

-   `@freenow/wave` version: 2.1.2

### Relevant code

```js
const container = document.querySelector('#root');
const root = createRoot(container!);
root.render(
    <>
        <ClassicColors />
        <LightScheme>
            <Root />
        </LightScheme>
    </>
);
```

```js
<PhoneInput listPortalTarget={document.body} />
```

### What was expected to happen?
Country code selection menu appears in light scheme colors.

### Reproduction
https://codesandbox.io/p/sandbox/wave-playground-forked-lxtsvk

**Comment by martimalek** [comment]
Hey @jahe, thanks for opening the issue! I've marked it as a bug 🐛 

If my understanding is correct what's happening is that `PhoneInput` uses `SelectList` underneath and when passing `listPortalTarget` to it you change where in the DOM the list will be rendered, in your case the `<body>`, since it is higher in the DOM than the `LightScheme` it won't have the scheme applied.

The solution will probably be to wrap the list in `LightScheme`/`DarkScheme` depending on the applied scheme on the component. We already have an internal hook that detects the applied scheme and even reacts to changes to it, so this part should be straightforward, the tricky bit will be how to wrap it because we are using `react-windowed-select` to render the list, so we are not directly creating the portal, we'd need to take a look at the code and check how flexible their API is, I don't recall exactly but I think we are able to pass our custom components to the library so that it renders them efficiently, maybe wrapping those components would do the trick.

I can try to take a look at it next week 👀

**Comment by JanHamara** [comment]
@jahe we will merge in the fix today, please in your use case use `menuPortalTarget` on PhoneInput instead of `listPortalTarget` 🙂

**Comment by martimalek** [comment]
Resolved by @JanHamara in #438
